src/Generator/Passes/CleanCommentsPass.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using CppSharp.AST;
+using CppSharp.Generators.CSharp;
+
+namespace CppSharp.Passes
+{
+    public class CleanCommentsPass : TranslationUnitPass,ICommentVisitor<bool>
+    {
+        public bool VisitBlockCommand(BlockCommandComment comment)
+        {
+            return true;
+        }
+
+        public override bool VisitDeclaration(Declaration decl)
+        {
+            if (!base.VisitDeclaration(decl))
+                return false;
+
+            if (decl.Comment != null)
+            {
+                var fullComment = decl.Comment.FullComment;
+                VisitFull(fullComment);
+                
+            }
+            return true;
+        }
+
+        public bool VisitFull(FullComment comment)
+        {
+            foreach (var block in comment.Blocks)
+                block.Visit(this);
+
+            return true;
+        }
+        #region Comments Visit
+        public bool VisitHTMLEndTag(HTMLEndTagComment comment)
+        {
+            return true;
+        }
+
+        public bool VisitHTMLStartTag(HTMLStartTagComment comment)
+        {
+            return true;
+        }
+
+        public bool VisitInlineCommand(InlineCommandComment comment)
+        {
+            return true;
+        }
+
+        public bool VisitParagraphCommand(ParagraphComment comment)
+        {
+            foreach (TextComment item in comment.Content)
+            {
+                if (item.Text.StartsWith("<", StringComparison.Ordinal) 
+                    && !item.Text.EndsWith(">", StringComparison.Ordinal))
+                    if (item.Text.Contains("summary"))
```

Comment:
We've already discussed that this should not be specific to "summary". It might be some other element. So please replace this check with something more general. For example, a regex which checks for a tag where a tag is a single word i.e. only Latin letters, digits and underscores and starts with a letter.